nixos/modules/tasks/filesystems/zfs.nix

```diff
@@ -203,18 +204,26 @@ in
 
   imports = [
     (mkRemovedOptionModule [ "boot" "zfs" "enableLegacyCrypto" ] "The corresponding package was removed from nixpkgs.")
+    # Introduce me in 24.05.
+    # (mkRemovedOptionModule [ "boot" "zfs" "enableUnstable" ] "`boot.zfs.enableUnstable` was replaced by `boot.zfs.package` where you can specify `pkgs.zfsUnstable` to track the unstable ZFS.")
```

Comment:
Can we put this into an extra commit and not backport it?

Reply:
Actually, we should use 24.11.